org.knime.knip.knimepython/src/org/knime/knip/knimepython2/ImgPlusDeserializerFactory.java

```diff
@@ -0,0 +1,92 @@
+package org.knime.knip.knimepython2;
```

Comment:
isn't this more or less a copy of https://github.com/knime-ip/knip-python-extensions/blob/master/org.knime.knip.knimepython/src/org/knime/knip/knimepython/ImgPlusSerializer.java? Maybe both classes should share some code?!